docs/userguides/clis.md

```diff
@@ -158,3 +158,23 @@ def create_account(alias):
     # We know the alias is not yet used in Ape at this point.
     click.echo(alias)
 ```
+
+You can control additional filtering of the alias account options by using one or both of the `account_type` or `account_list` kwargs.
+Use `account_type` to filter the choices by specific types of [AccountAPI](../methoddocs/api.html#ape.api.accounts.AccountAPI).
+Use the `account_list` kwarg to filter the choices by a given sequence of accounts.
+
+```python
+import click
+from ape import accounts
+from ape.cli import existing_alias_argument
+from ape_accounts.accounts import KeyfileAccount
+
+def _valid_accounts():
+   return [a for a in accounts if a.alias.startswith("root_signer_")]
```

Comment:
what's `root_signer_`?

Reply:
I was just trying to come up with made up example, I can make that more obvious though